src/components/calculators/Payslip.vue

```diff
@@ -1,549 +1,219 @@
-
-    }
-
-    // End of Output styles.
-
-    @media (min-width: 991px) {
-
-        // Start of Global responsive styles.
-
-        button {
-          width: auto;
-        }
-
-        .flex-grow {
-          flex-grow: 0;
-        }
-
-        .radio-spacing {
-          margin: 0 10px 0 0;
-        }
-
-        // End of Global responsive styles.
-
-        // Start of Input responsive styles.
-
-        .input-fields {
-            .form-group {
-                display: flex;
-                .input-field {
-                    border: 1px solid #dce6eb;
-                    padding: 0.6875rem;
-                }
-                .border-left {
-                  border-left: none;
-                }
-                .input-field--label {
-                    &:hover {
-                        background-color: #F2F2F2;
-                    }
-                    &:active {
-                        background-color: #F2F2F2;
-                    }
-                }
-                .input-field--number {
-                    width: 35%;
-                }
-            }
-            .form-group--mobile {
-              .input-field--label {
-                width: auto;
-                margin-bottom: 0;
-              }
-            }
-            .form-group--twos {
-                width: 80%;
-                .radio-spacing {
-                    width: 47%;
-                    margin: 0 10px 10px 0;
-                }
-            }
-            .form-group--loan-twos {
-              width: 90%;
-            }
-        }
-
-        .input--width {
-            width: 50%;
-            display: inline-block;
-        }
-
-        // End of Input responsive styles.
-
-        // Start of Output responsive styles.
-
-        .output--width {
-           width: 50%;
-            display: inline-block;
-            float: right;
-        }
-
-        .tabs-component {
-          border: 0;
-        }
-
-        .tabs-component-tab {
-            background-color: #fff;
-            border: solid 1px #ddd;
-            transform: translateY(2px);
-            transition: transform .3s ease;
-            width: 34%;
-        }
-
-        .tabs-component-tabs {
-            align-items: stretch;
-            display: flex;
-            justify-content: flex-start;
-            margin-bottom: -1px;
-            padding: 0;
-        }
-
-        .tabs-component-tab.is-active {
-            border-bottom: solid 1px #f2f2f2;
-            z-index: 2;
-            transform: translateY(0);
-            border-top: 0.25em solid #1380bc;
-        }
-
-        .tabs-component-panels {
-            border-top-left-radius: 0;
-            background-color: #fff;
-            border: solid 1px #ddd;
-            border-radius: 0 6px 6px 6px;
-            box-shadow: 0 0 10px rgba(0, 0, 0, .05);
-        }
-
-        // End of Output responsive styles.
-
-    }
-</style>
+  @import "../../scss/globals.scss";
+</style>
```

Comment:
Good work @kevin-chilcott-sift just think about the border modifier and legibility of code 👍 

This beast of a calculator got tamed with BEM 🥇